vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageTextItem.kt

```diff
@@ -92,27 +91,22 @@ abstract class MessageTextItem : AbsMessageItem<MessageTextItem.Holder>() {
                 it.bind(holder.messageView)
             }
         }
-        val textFuture = if (bindingOptions?.canUseTextFuture.orFalse()) {
-            PrecomputedTextCompat.getTextFuture(
-                    message?.charSequence ?: "",
-                    TextViewCompat.getTextMetricsParams(holder.messageView),
-                    null)
-        } else {
-            null
-        }
-        markwonPlugins?.forEach { plugin -> plugin.beforeSetText(holder.messageView, message?.charSequence as Spanned) }
```

Comment:
also noticed `message?.charSequence as Spanned` will throw if the message is null, will wrap with a `.let`

Reply:
Yes, I'm pretty sure `message` cannot be null but you are right, better safe than sorry.